connector/xconnector/connector.go

```diff
@@ -57,86 +57,24 @@ type Profiles interface {
 // CreateTracesToProfilesFunc is the equivalent of Factory.CreateTracesToProfiles().
 type CreateTracesToProfilesFunc func(context.Context, connector.Settings, component.Config, xconsumer.Profiles) (connector.Traces, error)
 
-// CreateTracesToProfiles implements Factory.CreateTracesToProfiles().
-// Deprecated: [v0.120.0] No longer used, will be removed.
-func (f CreateTracesToProfilesFunc) CreateTracesToProfiles(ctx context.Context, set connector.Settings, cfg component.Config, next xconsumer.Profiles) (connector.Traces, error) {
-	if f == nil {
-		return nil, internal.ErrDataTypes(set.ID, pipeline.SignalTraces, xpipeline.SignalProfiles)
-	}
-	return f(ctx, set, cfg, next)
-}
-
 // CreateMetricsToProfilesFunc is the equivalent of Factory.CreateMetricsToProfiles().
 type CreateMetricsToProfilesFunc func(context.Context, connector.Settings, component.Config, xconsumer.Profiles) (connector.Metrics, error)
 
-// CreateMetricsToProfiles implements Factory.CreateMetricsToProfiles().
-func (f CreateMetricsToProfilesFunc) CreateMetricsToProfiles(ctx context.Context, set connector.Settings, cfg component.Config, next xconsumer.Profiles) (connector.Metrics, error) {
```

Comment:
Did this one get missed last time around? I don't see the deprecation notice on it

Reply:
Oops, looks like I missed this one yeah :( What should we do? We could wait one more version for this method if you think that's better. 

When it comes to open source usages I cannot find any for this function ([search](https://github.com/search?q=language%3AGo+CreateMetricsToProfiles+-is%3Afork+-repo%3Aopen-telemetry%2Fopentelemetry-collector+-repo%3A2lambda123%2Fopen-telemetry-opentelemetry-collector++-repo%3Arpradeepkumar7%2Ffirst-opentelemetry-exporter&type=code), excludes a couple fork-like repos that are not actually forks), so IMO it should be fine to go ahead

Reply:
it's probably ok to remove in this PR, and maybe just add a note to the changelog specifically for this one?

Reply:
It is in xconnector package, so experimental is ok to change in the current version.

Reply:
Added a note, thanks for catching this!